cmd/cluster/logs/cluster.go

```diff
@@ -0,0 +1,111 @@
+package logs
+
+import (
+	"fmt"
+	sdk "github.com/openshift-online/ocm-sdk-go"
+	v1 "github.com/openshift-online/ocm-sdk-go/clustersmgmt/v1"
+	ocmutils "github.com/openshift/osdctl/pkg/utils"
+	"os"
+)
+
+func fetchDetails(clusterKey string) (string, string, string) {
+	if err := ocmutils.IsValidClusterKey(clusterKey); err != nil {
+		fmt.Println(err)
+		os.Exit(1)
```

Comment:
**chore (repeating)**: Can we bubble up the error instead of manually calling exit here?